workspaces/optic-engine/src/interactions/visitors/diff.rs

```diff
@@ -197,26 +197,36 @@ impl RequestBodyVisitor<InteractionDiffResult> for DiffRequestBodyVisitor {
   fn visit(&mut self, interaction: &HttpInteraction, context: &RequestBodyVisitorContext) {
     if let Some(operation) = context.operation {
       let actual_content_type = &interaction.request.body.content_type;
+      let actual_body: Option<BodyDescriptor> = (&interaction.request.body.value).into();
       let (request_id, request_body_descriptor) = operation;
```

Comment:
Nit on a bit of possible clean up while we're touching this: I wonder if we can rename these to be a little bit less confusing, I had to look at this multiple times to see what was what 😅. Took me a bit to realise`actual_body` is actually a `BodyDescriptor` not a `Body`, while the thing called `request_body_descriptor` isn't a `BodyDescriptor`. "actual" is also pretty vague, come to think of it; actually what? Perhaps distinguishing between what comes from the interaction and what comes from the spec is helpful? 🤷‍♂️

Reply:
Changed to `interaction_...` and `maybe_...` to indicate `Option`